stable/spinnaker/values.yaml

```diff
@@ -2,7 +2,7 @@ halyard:
   spinnakerVersion: 1.16.1
```

Comment:
Would it make sense to also bump Spinnaker to either the latest patch 1.16.7 or the latest minor 1.18.5?
Or should that go in a different PR?